code.js

```diff
@@ -40,10 +40,23 @@ function getTheWeather() {
     .then(data => {
       return data.json();
     }) //Returns a promise
-    .then(res => {
-      //Gives promise a handle (res)
+    .then(res => { //Gives promise a handle (res)
+      console.log(res);
+
+      //Display local temperature
       let fTemp = ((res.main.temp - 273.15) * 9) / 5 + 32; //Converts JSON object element res.main.temp from K° to F°
       document.querySelector("#theTemp").innerHTML = Math.round(fTemp); //Rounds and displays current temperature
+      console.log(`fTemp: ${fTemp}`);
+
+      //Display weather image
+      let iconUrl = `https://openweathermap.org/img/wn/${res.weather[0].icon}@2x.png`;
+      document.getElementById("weather-image").src = iconUrl;
+      console.log(`iconUrl: ${iconUrl}`);
```

Comment:
I'd recommend moving this block of code to a helper function, then calling it from within `getTheWeather()`. At the same time, refactor L46-L48 into a helper function too. We don't want to get the logic from two different functions mixed up together.